testuseradd.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2016 Red Hat Inc.
+# Satyajit Bulage<bulge.satyajit007@gmail.com>
+#
+# This program is a free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 2 of the license, or(at your option) any
+# later version. See http://www.gnu.org/copyleft/gpl.html for the full text of
+# the license.
+#
```

Comment:
We are yet to add the official license to the tests, so that has to be happen first, and also we need those to be in GPLv3 preferably.